docs/dev/table/ddl.md

```diff
@@ -0,0 +1,230 @@
+---
+title: "DDL"
```

Comment:
I'm not sure about this. Should we put DDL into `dev/table/sql.md` as a DDL section? Because it's strange to have DDL in the same level with "Table API" and "SQL".

Reply:
I agree with Jark. Either we put it in the SQL page or split the SQL page into more subpages. I would vote for the first approach.